### PR TITLE
FS-60 used different tutorial, this oauth works from Windows side

### PR DESCRIPTION
**Pull request by lestgabo**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -32,9 +32,9 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        // TODO: Specify your own unique Application ID (https://Adeveloper.android.com/studio/build/application-id.html).
```
Comment (Aldbas) [inline android/app/build.gradle:35]: i think you added extra 'A' in that 'Adeveloper.android.com' 😄 😄 😸

File: pubspec.lock [hunk 2]
```diff
@@ -123,6 +123,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.2"
+  rxdart:
```
Comment (Aldbas) [inline pubspec.lock:126]: what's this be? I've never touched this file. what it do?

**Review by Aldbas — APPROVED** [review]
yup niceu can you implment in ishallsealtheheavens

File: pubspec.lock (hunk 2 above)
Comment (lestgabo) [reply pubspec.lock:126]: rxdart is an extension of Stream -> allows you to do streams easier and more understandably
https://medium.com/flutter-community/why-use-rxdart-and-how-we-can-use-with-bloc-pattern-in-flutter-a64ca2c7c52d

but in this case its just used for its `Obersvable<T>`

File: android/app/build.gradle (hunk 1 above)
Comment (lestgabo) [reply android/app/build.gradle:35]: wow lucky it was just in a TODO

**Pull request merged**